Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/Cartridges/heavy_rifle.yml

```
@@ -20,11 +20,69 @@
+  parent: LPPBaseCartridgeMinigun
+  components:
+  - type: CartridgeAmmo
+    proto: BaseBulletPractice
+    deleteOnSpawn: true
+
+- type: entity
+  id: LPPCartridgeMinigunAP
+  name: патрон (.10 Винтовочный бронебойные)
+  parent: LPPBaseCartridgeMinigun
+  components:
+  - type: CartridgeAmmo
```

yamllint

[error] wrong indentation: expected 4 but found 2

(indentation)